doc/developers/demo.rst

```diff
@@ -1,76 +1,129 @@
 End-to-end demo
 ===============
 
-This document explains how the Python testing infrastructure is used to run a complete end-to-end test of CCF.
+This document explains how to spin up a test CCF network and submit simple commands to it using `curl`_. We use curl here because it is basic, general, and broadly available - you should be able to get the same results with any HTTP client, provided you can configure the appropriate caller and CA identities. Once you have built your own app, you should be able to test it in the same way - simply replace ``liblogging`` with the name of your app binary, and call the endpoints defined by your app.
```

Comment:
I would drop the word basic ==> consider: "we use curl, an industry stand tool, " ...